packages/angular/src/utils/versions.ts

```diff
@@ -23,7 +23,7 @@ export const postcssUrlVersion = '~10.1.3';
 export const autoprefixerVersion = '^10.4.0';
 export const tsNodeVersion = '10.9.1';
 
-export const jestPresetAngularVersion = '~12.2.3';
+export const jestPresetAngularVersion = '~13.0.0-next.1';
```

Comment:
Is it safe for us to be depending on a pre-release version here?

Reply:
I'm just testing here waiting for v13 stable release